Add docs/overlays for aws arn annotation (#137)
vmware-tanzu/cartographer-conventions: README.md

    Note: you'll need to `export KO_DOCKER_REPO=<ACCESSIBLE_DOCKER_REPO>` such that `ko` can push to the repository and your cluster can pull from it. Visit [the ko README](https://github.com/google/ko/blob/master/README.md#usage) for more information.

> ## Running cartographer convention on an AWS

Reviewer: ```suggestion
## Running cartographer convention on an AWS cluster
```

> If you want to [attach an IAM role](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to the service account that the controller uses, you can provide the role arn during the time of installation

Reviewer: ```suggestion
In order to [attach an IAM role](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to the service account that the controller uses, provide the role [arn](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) during installation phase.
```
